taskcluster/xpi_taskgraph/transforms/post_build.py

```diff
@@ -55,11 +55,16 @@ def test_tasks_from_manifest(config, tasks):
         env["ARTIFACT_PREFIX"] = artifact_prefix
         if xpi_config.get("install-type"):
             env["XPI_INSTALL_TYPE"] = xpi_config["install-type"]
-        paths = []
-        for artifact in xpi_config["artifacts"]:
-            artifact_name = f"{artifact_prefix}/{os.path.basename(artifact)}"
-            paths.append(artifact_name)
-        upstreamArtifacts = [{"taskId": "<build>", "paths": paths}]
-        env["XPI_UPSTREAM_URLS"] = json.dumps(upstreamArtifacts)
+
+        if task["label"].startswith("addons-linter-"):
```

Comment:
Maybe go with `config.kind`. But even still, maybe we can add a generic `task.pop("only-for-formats")`:

```python
if task.get("only-for-formats"):
    if xpi_config.get("addon-type") not in task.pop("only-for-formats"):
        continue
```

Then in the linter we'd add `only-for-formats: ["privileged"]`, and we'd have a generic way of limiting certain tests to a set of formats.

Reply:
Yeah, that's a better solution, thanks!